python/pyarrow/tests/test_dataset.py

```diff
@@ -1096,6 +1193,79 @@ def test_open_dataset_from_uri_s3(s3_connection, s3_server):
     assert dataset.to_table().equals(table)
 
 
+@pytest.mark.parquet
+@pytest.mark.s3
+def test_open_dataset_pina(s3_connection, s3_server):
```

Comment:
What is "pina"? 

(also, part of this test seems to duplicate the test above?)

Reply:
Sorry, that a quick placeholder I use. Updating.